en/Plugins/Graph view.md

```diff
@@ -72,3 +72,9 @@ This section controls the forces that act on each node in the graph.
 Notes and attachments appear in chronological order based on their creation time.
 
 ![[Pasted image 10.png]]
+
+## Local Graph
+
+To open a local Graph view, use the **Open local graph** command. While the Graph view shows all notes in your vault, a local Graph view shows you notes connected to the active note.
```

Comment:
```suggestion
While the graph view shows all notes in your vault, you can also open a local graph view that shows you notes connected to the active note.

To open a local Graph view, use the **Open local graph** command. 
```

Flipping the sentence so it introduces the feature before explaining how to use it.